src/components/Activity/ActivityItem.js

```diff
@@ -0,0 +1,242 @@
+import React, { useCallback } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ButtonBase,
+  ButtonIcon,
+  IconCross,
+  IconCheck,
+  GU,
+  blockExplorerUrl,
+  textStyle,
+  useTheme,
+  IdentityBadge,
+} from '@aragon/ui'
+import { transformAddresses, getNetworkType } from '../../lib/web3-utils'
+import {
+  ACTIVITY_STATUS_PENDING,
+  ACTIVITY_STATUS_CONFIRMED,
+  ACTIVITY_STATUS_FAILED,
+  ACTIVITY_STATUS_TIMED_OUT,
+} from './activity-statuses'
+import TransactionProgress from './TransactionProgress'
+import { useActivity } from './ActivityProvider'
+import { getActivityData } from './activity-types'
+import TimeTag from './TimeTag'
+
+function ActivityItem({ activity }) {
+  const theme = useTheme()
+  const { removeActivity } = useActivity()
+
+  const activityData = getActivityData(
+    activity.activityType,
+    activity.activityParams
+  )
+
+  const handleOpen = useCallback(() => {
+    if (activity.transactionHash) {
+      window.open(
+        blockExplorerUrl('transaction', activity.transactionHash, {
+          networkType: getNetworkType(),
```

Comment:
In this case we should use `useNetwork` hook that does the `mainnet` to `main` convertion.

Reply:
This made me notice that we were a bit inconsistent with the network ID:

- `getNetworkId()` was returning `mainnet` rather than `main` which is more standard.
- `useNetwork()` was only doing this conversion, and doesn’t really need to be a hook.
- In some cases, we were using `getNetworkName()` to get an idea rather than `getNetworkId()` or `useNetwork()`.

So I removed `useNetwork()` and made everything else use `main`, see https://github.com/aragon/court-dashboard/pull/100/commits/2e46f755e6b2550825b7c7cab1c1ae51dbf89098

Let me know what you think!

Reply:
LGTM!